lib/utils/patterns.ts

```diff
@@ -253,6 +253,53 @@ export function awaitDevice(uuid: string) {
 	});
 }
 
+export async function awaitDeviceOsUpdate(
```

Comment:
Just an observation: this `async` function does not have any `await` clauses...

Reply:
Nice spot, I will remove it and squash all commits.